fs/accounting/stats.go

```diff
@@ -558,8 +551,9 @@ func (s *StatsInfo) RetryAfter() time.Time {
 
 // NewCheckingTransfer adds a checking transfer to the stats, from the object.
 func (s *StatsInfo) NewCheckingTransfer(obj fs.Object) *Transfer {
-	s.checking.add(obj.Remote())
-	return newCheckingTransfer(s, obj)
+	tr := newCheckingTransfer(s, obj)
```

Comment:
What do all these changes do? They don't appear to be sorting transfers by start time? Do they need to go into another commit?

Reply:
No. This have to do with stringSet -> transferMap changes. If you need more commits, I could seperate them.

Reply:
Can you separate them, please? I'd like to have one commit with just the sorting changes in just in case I need to revert it.

Thank you :-)

Reply:
Or should I just put all stringSet->transferMap in the first commit?